app/views/RoomActionsView/index.js

```diff
@@ -667,117 +391,465 @@ class RoomActionsView extends React.Component {
-				accessibilityTraits='button'
-				enabled={!item.disabled}
-				testID={item.testID}
-				theme={theme}
-			>
-				<View style={styles.sectionItem}>
-					{subview}
-				</View>
-			</Touch>
+			<List.Section>
+				<List.Separator />
+				<List.Item
+					title='Voice_call'
+					onPress={() => RocketChat.callJitsi(room?.rid, true)}
+					testID='room-actions-voice'
+					left={() => <List.Icon name='phone' />}
+					showActionIndicator
+				/>
+				<List.Separator />
+				<List.Item
+					title='Video_call'
+					onPress={() => RocketChat.callJitsi(room?.rid)}
+					testID='room-actions-video'
+					left={() => <List.Icon name='camera' />}
+					showActionIndicator
+				/>
+				<List.Separator />
+			</List.Section>
 		);
 	}
 
-	renderItem = ({ item }) => {
-		const { theme } = this.props;
-		const colorDanger = { color: themes[theme].dangerColor };
-		const subview = item.type === 'danger' ? (
-			<>
-				<CustomIcon name={item.icon} size={24} style={[styles.sectionItemIcon, colorDanger]} />
-				<Text style={[styles.sectionItemName, colorDanger]}>{ item.name }</Text>
-			</>
-		) : (
-			<>
-				<CustomIcon name={item.icon} size={24} style={[styles.sectionItemIcon, { color: themes[theme].bodyText }]} />
-				<Text style={[styles.sectionItemName, { color: themes[theme].bodyText }]}>{ item.name }</Text>
-				{item.description ? <Text style={[styles.sectionItemDescription, { color: themes[theme].auxiliaryText }]}>{ item.description }</Text> : null}
-				{item?.right?.()}
-			</>
-		);
-		return this.renderTouchableItem(subview, item);
+	renderE2EEncryption = () => {
+		const {
+			room, canEdit
+		} = this.state;
+		const { e2eEnabled } = this.props;
+
+		// If can edit this room
+		// If this room type can be Encrypted
+		// If e2e is enabled for this server
+		if (canEdit && E2E_ROOM_TYPES[room?.t] && e2eEnabled) {
+			return (
+				<List.Section>
+					<List.Separator />
+					<List.Item
+						title='Encrypted'
+						testID='room-actions-encrypt'
+						left={() => <List.Icon name='encrypted' />}
+						right={this.renderEncryptedSwitch}
+					/>
+					<List.Separator />
+				</List.Section>
+			);
+		}
+		return null;
 	}
 
-	renderSectionSeparator = (data) => {
+	renderLastSection = () => {
+		const { room, joined } = this.state;
 		const { theme } = this.props;
-		if (data.trailingItem) {
-			return <View style={[styles.sectionSeparator, data.leadingSection && styles.sectionSeparatorBorder, { backgroundColor: themes[theme].auxiliaryBackground, borderColor: themes[theme].separatorColor }]} />;
+		const { t, blocker } = room;
+
+		if (!joined || t === 'l') {
+			return null;
 		}
-		if (!data.trailingSection) {
-			return <View style={[styles.sectionSeparatorBorder, { backgroundColor: themes[theme].auxiliaryBackground, borderColor: themes[theme].separatorColor }]} />;
+
+		if (t === 'd') {
+			return (
+				<List.Section>
+					<List.Separator />
+					<List.Item
+						title={`${ blocker ? 'Unblock' : 'Block' }_user`}
```

Comment:
This icon doesn't exist

Reply:
It's working on this PR, looks like it just doesn't exists on the new icons selection updated on #2567.